### PR TITLE
Create dev container for Sinergym environment

### PR DESCRIPTION
**Pull request by dylan-moss**
Allow for mounting of local files within Docker container for ease of development with VS Code.

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
+	"name": "hnp",
+	"build": {
+		"args": {
+			"ENERGYPLUS_VERSION": "9.5.0",
+			"ENERGYPLUS_INSTALL_VERSION": "9-5-0",
+			"ENERGYPLUS_SHA": "de239b2e5f"
+		}
+	},
+	"dockerFile": "../Dockerfile",
```
Comment (XkunW) [inline .devcontainer/devcontainer.json:10]: Seems like this is using the Dockerfile in the parent dir, if so, maybe remove the Dockerfile in .devcontainer?

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+				"icon": "terminal-tmux"
+			},
+			"pwsh": {
+				"path": "pwsh",
+				"icon": "terminal-powershell"
+			}
+		},
+		"terminal.integrated.defaultProfile.linux": "bash"
+	},
+	"extensions": [
```
Comment (XkunW) [inline .devcontainer/devcontainer.json]: Are all the following extensions needed?

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (dylan-moss) [reply .devcontainer/devcontainer.json:10]: Removed it now.

File: .devcontainer/devcontainer.json (hunk 2 above)
Comment (dylan-moss) [reply .devcontainer/devcontainer.json]: No these were just what was being used in the Sinergym dev container, but I have removed them now.

**Review by XkunW — APPROVED** [review]

**Pull request merged**